### PR TITLE
doc pytest

### PR DESCRIPTION
**Pull request by ntyukaev**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Review by andrew-zaytsev — APPROVED** [review]

**Comment by ntyukaev** [comment]
@azhogov could you please review this mr?

File: .github/workflows/build_doc.yml [hunk 1]
```diff
@@ -33,15 +42,32 @@ jobs:
...
+                 --html="./build/docs/_artifacts/doc-generation.html" \
+                 --doxygen-strip="$(pwd)" \
+                 --doxygen-xfail="./docs/doxygen-xfail.txt" \
+                 --self-contained-html ./docs/scripts/tests/test_docs.py
+
+      - name: 'Upload html'
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: openvino_html
```
Comment (azhogov) [inline .github/workflows/build_doc.yml:65]: it is better to pack(zip) before uploading (speed up)

File: docs/scripts/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
+""" Configuration for tests.
```
Comment (azhogov) [inline docs/scripts/tests/conftest.py:1]: Add a license

File: docs/scripts/tests/utils/layout.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+from collections import defaultdict
```
Comment (azhogov) [inline docs/scripts/tests/utils/layout.py:1]: add a purpose comment

**Review by azhogov — APPROVED** [review]

**Pull request merged**